src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/JndiDirectoryDAO.scala

```diff
@@ -73,12 +119,13 @@ class JndiDirectoryDAO(protected val directoryConfig: DirectoryConfig)(implicit
 
   override def loadGroup(groupName: SamGroupName): Future[Option[SamGroup]] = withContext { ctx =>
     Try {
-      val attributes = ctx.getAttributes(groupDn(groupName), Array(Attr.cn, Attr.member))
+      val attributes = ctx.getAttributes(groupDn(groupName))
 
       val cn = getAttribute[String](attributes, Attr.cn).getOrElse(throw new WorkbenchException(s"${Attr.cn} attribute missing: $groupName"))
+      val email = getAttribute[String](attributes, Attr.email).getOrElse(throw new WorkbenchException(s"${Attr.email} attribute missing: $groupName"))
```

Comment:
difference between Attr.email and groupName? if groupName is: ask, then Attr.email is GROUP_ask@gmail.com?